integration/benchmark/cmd/ci/main.go

```diff
@@ -103,9 +103,6 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	sp := benchmark.NewStatsPusher(ctx, log, pushgateway, "loader", prometheus.DefaultGatherer)
```

Comment:
why was this deleted?  do you still need it for non-localDev mode?

Reply:
When running in CI we are not using prometheus and because of how the self-hosted runner works actually don't run the prometheus node at all (see `start-thin` in the Makefile). If we leave it in then the output is full of warnings instead of our data. This still exists in the manual version of the loader for when local testing is needed and prometheus is actually accessible.

Reply:
Do we have a Prometheus setup for this environment?  Can we just send the data there?

Reply:
(If there is no Prometheus setup there, you can consider hiding it behind the same `localDev` flag.)

Reply:
There is no Prometheus on the CI environment, will hide this behind the localDev flag.